### PR TITLE
SUS-646: Return user ids

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
https://wikia-inc.atlassian.net/browse/SUS-646

### CODE REVIEW

**Comment by lukaszkonieczny** [comment]
cc @macbre @mixth-sense

File: maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php [hunk 1]
```diff
@@ -28,7 +28,9 @@ public function execute() {
 			->SELECT( 'wup_user' )
 			->FROM( 'wikia_user_properties' )
 			->WHERE( 'wup_property' )->EQUAL_TO( 'editcountThisWeek' )
-			->run( $dbw );
+			->runLoop( $dbw, function( &$userIds, $row ){
+				$userIds[] = $row->wup_user;
+			} );
```
Comment (macbre) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php]: You can use `Database::selectFieldValues` here.

File: maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php [hunk 2]
```diff
@@ -24,16 +24,17 @@ public function execute() {
...
-
-		$result = ( new WikiaSQL() )
-			->DELETE( 'wikia_user_properties' )
-			->WHERE( 'wup_property' )->EQUAL_TO( 'editcountThisWeek' )
-			->run( $dbw );
+		$userIds = $dbw->selectFieldValues(
+			'wikia_user_properties',
+			'wup_user',
+			[ 'wup_property' => 'editcountThisWeek' ]
+		);
```
Comment (macbre) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php:32]: `__METHOD__` is missing here

Comment (lukaszkonieczny) [inline maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php:32]: it's default value

**Pull request merged**